src/app/other-user-skills/other-user-skills.component.ts

```diff
@@ -0,0 +1,68 @@
+import { ChangeDetectorRef, Component, OnInit } from '@angular/core';
+import { UserSkillsService } from '../user-skills/user-skills.service';
+import { ActivatedRoute } from '@angular/router';
+import { map } from 'rxjs/operators';
+import { UsersService } from '../users/users.service';
+import { User } from '../users/user';
+import { GlobalErrorHandlerService } from '../error/global-error-handler.service';
+import { UserSkillView } from '../skill-card/user-skill-view';
+
+@Component({
+  selector: 'app-other-user-skills',
+  templateUrl: './other-user-skills.component.html',
+  styleUrls: ['./other-user-skills.component.scss']
+})
+export class OtherUserSkillsComponent implements OnInit {
+
+  userSkills: UserSkillView[] = [];
+  user: User = null;
+  errorMessage: string = null;
+
+  constructor(private userSkillsService: UserSkillsService,
+              private userService: UsersService,
+              private activatedRoute: ActivatedRoute,
+              private changeDetector: ChangeDetectorRef,
+              private globalErrorHandlerService: GlobalErrorHandlerService) {
+  }
+
+  ngOnInit() {
+    this.activatedRoute.paramMap
+      .pipe(map(params => params.get('userId')))
+      .subscribe(userId => {
+        this.loadUser(userId);
+      });
+  }
+
+  private loadUser(userId: string) {
+    this.userService.getUserById(userId)
```

Comment:
Can we use [UserIdentityService.getUserIdentity](https://github.com/KnowledgeAssets/myskills-webapp/blob/master/src/app/shared/user-identity.service.ts#L18) instead? It seems to me that we need user identifier, user first name and user last name in the component. All of them can be fetched from [UserIdentity](https://github.com/KnowledgeAssets/myskills-webapp/blob/master/src/app/shared/user-identity.ts). Will that do?